ext/handlebars/src/main/java/org/eclipse/krazo/ext/handlebars/DefaultHandlebarsProducer.java

```diff
@@ -29,11 +32,18 @@
  * @author Christian Kaltepoth
  */
 public class DefaultHandlebarsProducer {
+    @Inject
+    private ServletContext servletContext;
 
     @Produces
     @ViewEngineConfig
     public Handlebars getHandlebars() {
-        return new Handlebars();
+        Handlebars handlebars = new Handlebars();
+        
+        // setup TemplateLoader to allow for finding partials
+        handlebars.with(new ServletContextTemplateLoader(servletContext, ViewEngine.DEFAULT_VIEW_FOLDER));
```

Comment:
Can you please use the `ViewEngine#VIEW_FOLDER` here, so a possibly customized Folder is used? You find an example on how to load the property in `ViewEngineBase` line 48

Reply:
@erdlet let me know if my changes make sense. I added a basic partial test to the existing Handlebars template test. One comment I would add is that it would be nice to get some documentation around how to run the individual tests. I could probably sit down and figure out what profiles to run, but it would be helpful to have a quick example in the README.

Reply:
Thanks for that feedback! You're completely right, this needs to be documented. I hope to find some time for that in the next few days. Anyway, if you want, you can open an issue for that, so it isn't forgotten in case I have no time.

Reply:
The code seems senseful to me, thanks for the fix of the view folder.